XR.Mono.Cover/CoverHost.cs

```diff
@@ -157,7 +157,7 @@ void MarkType( TypeMirror t )
 
             loadedTypes.Add( t.FullName );
 
-            var meths = t.GetMethods ();
+            var meths = t.GetMethods ().Where ( m => m.IsPublic);
```

Comment:
I don't think I can accept this specific change.

It seems very odd to only cover public methods.  That is not generally what coverage is aimed at.

Reply:
If anything, the viewer or result renderer will be the place where you want to filter what gets counted, not in the coverage collection system

Reply:
well unit tests are only supposed to be used on public methods.   that's their reason d'etre.

however do agree code coverage can have other purposes test coverage.   

perhaps it should be added as a flag to covem? something like  --public-only?